Libraries/LibWeb/HTML/HTMLScriptElement.cpp

```diff
@@ -641,6 +641,27 @@ void HTMLScriptElement::unmark_as_parser_inserted(Badge<DOM::Range>)
     m_parser_document = nullptr;
 }
 
+// https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflecting-content-attributes-in-idl-attributes
+TrustedTypes::TrustedScriptURLOrString HTMLScriptElement::src() const
```

Comment:
The regression was due to losing the URL extended attribute from the IDL in f65dca1b5334bde716dc45a0c90659ebe36a7f22. Since that was originally added as a non standard attribute, this is now specified as ReflectURL in the HTML spec here: https://html.spec.whatwg.org/multipage/scripting.html#the-script-element:xattr-reflecturl

Ideally, the fix here would be just adding back that extended attribute. However, the IDL from the trusted types spec isn't updated there, they probably need to do an update. I am also not sure if our IDL generator supports that reflection since the trusted types type is no longer just a USVString

Reply:
I don't think our IDL generator supports that, at least I get some errors locally trying to make it Reflect URL. What should I do to make this PR mergable? Fix IDL generator and move it there?

Reply:
I added the comment

Reply:
To add to this I think we should also open an issue in https://github.com/w3c/trusted-types reflecting that their partial interface in https://github.com/w3c/trusted-types/blob/fbd1144e4c1978b7927d93b763b16fe8a569a657/spec/index.bs#L1283 should include the `ReflectURL` attribute